python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1140,13 +1135,10 @@ async def do_disableCorrection(self, data: type_hints.BaseMsgType) -> None:
         self.mark_corrections(data, False)
         await asyncio.sleep(0.0)  # give control back to event loop
 
-        # Lower mirrors if appropriate
-        if data == "disableAll":
-            await self.lower_mirrors_to_hardpoints(m1=True, m2=True)
-        elif data == "m1":
-            await self.lower_mirrors_to_hardpoints(m1=True, m2=False)
-        elif data == "m2":
-            await self.lower_mirrors_to_hardpoints(m1=False, m2=True)
+        await self.lower_mirrors_to_hardpoints(
```

Comment:
This would benefit from a unit test if possible. However, release and deploy an alpha for the time being.

Reply:
I updated the enable correction unit tests to also check the disable correction. Also took the opportunity to make a few additional updates to the CSC to future proof it against some features marked for deprecation in salobj.